packages/next-plugin-material-ui/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@next/plugin-material-ui",
+  "version": "9.1.2-canary.1",
+  "nextjs": {
+    "name": "Material UI",
+    "required-env": []
+  },
+  "peerDependencies": {
+    "next": "*"
```

Comment:
Is `@material-ui/styles` supposed to be a peer dependency as well?

Reply:
Yes, update. Although this is just an example `material-ui` plugin for testing